doc/dev/minikube.md

```diff
@@ -142,7 +142,7 @@ This has to happen before deploying kubecf.
 ```sh
 helm install stable/nginx-ingress \
   --name ingress \
-  --namespace ingress
+  --namespace ingress \
```

Comment:
Cross check, does `doc/dev/general.md` have the same missing backslash ?

Reply:
I thought this was fixed already... weird. Thanks for fixing it.